packages/react-graphql-universal-provider/src/GraphQLUniversalProvider.tsx

```diff
@@ -1,33 +1,64 @@
 import React from 'react';
 
+import ApolloClient, {ApolloClientOptions} from 'apollo-client';
 import {useSerialized} from '@shopify/react-html';
-import {ApolloProvider, createSsrExtractableLink} from '@shopify/react-graphql';
+import {
+  ApolloProvider,
+  createSsrExtractableLink,
+  createOperationDetailsLink,
+  GraphQLOperationDetails,
+} from '@shopify/react-graphql';
 import {useLazyRef} from '@shopify/react-hooks';
 import {createApolloBridge} from '@shopify/react-effect-apollo';
+import {useNetworkManager} from '@shopify/react-network';
+
+export type GraphQLClientOptions = ApolloClientOptions<any>;
+export const GRAPHQL_OPERATIONS = Symbol('graphQLOperations');
```

Comment:
I considered having this in `@shopify/react-graphql` at one point. I ended up leaving it here since `GraphQLUniversalProvider` does the _handshaking_ between the `@shopify/react-graphql` & `@shopify/react-network`.